src/client/core-client.ts

```diff
@@ -63,11 +67,14 @@ interface StopServiceReply {
 }
 
 interface DeployServiceRequest {
-    service: Service
+    url: string
+    chunk: any
```

Comment:
i think this can be updated as uint8 array but i'm not sure if bytes are encoded in 8bit by protobuf

Reply:
same here can be another pr if needed